src/components/Gallery.js

```diff
@@ -79,11 +79,24 @@ const LoadingAbout = () => {
 
 export const Gallery = ({ close, closeAll, items, startFileName, lang }) => {
   const i18n = useI18n()
+  const containerRef = useRef()
   const [
     currentIndex, onPrevImage, onNextImage
   ] = useRange(getInitialIndex(items, startFileName), items.length - 1)
   const [showAboutPopup] = usePopup(AboutContainer, { stack: true })
 
+  const onImageLoad = ({ target: img }) => {
+    const galleryNode = containerRef.current
+    const galleryClasses = ['hasHeader', 'portrait', 'landscape']
+
+    galleryClasses.forEach(galleryClass => {
+      galleryNode.classList.remove(galleryClass)
+    })
+
+    img.height >= img.width ? galleryNode.classList.add('portrait') : galleryNode.classList.add('landscape')
+    if (items[currentIndex].caption) galleryNode.classList.add('hasHeader')
```

Comment:
I know this is not enforced everywhere but please use the form with braces even for one-liner

```
if (...) {
  ...
}
```